frontend/admin/src/js/components/poolCandidate/CreatePoolCandidate.tsx

```diff
@@ -173,13 +172,13 @@ const UserFormSection: React.FunctionComponent<{
             defaultMessage: "Telephone",
             description: "Label displayed on the user form telephone field.",
           })}
-          type="tel"
+          type="text"
```

Comment:
Why did you switch from `tel` to `text`?  Seems like that would be useful even without the pattern?